src/shared/api.js

```diff
@@ -0,0 +1,39 @@
+const getAllVenues = () => {
+  return fetch(`/http://localhost:3000/venues/`, {
+    method: "GET",
+  })
+    .then((res) => res.json())
+    .catch((error) => console.error("Error fetching venues:", error));
```

Comment:
this kind of error is not any feedback a user can see. You should use a MUI Snackbar in case of any errors so user can be notified about errors. 
 
 To implement snackbar you need to 
 1. render snackbar at the top level component (App.jsx or Layout) based on the state
 2. The state can be kept in Context
 3. Use the context setter whenever you need the Snackbar to be triggered

Reply:
done